components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceEndpointConfig.java

```diff
@@ -143,6 +147,12 @@
     @UriParam
     private String instanceId;
 
+    // Streaming API properties
+    @UriParam
+    private Integer defaultReplayId;
+    @UriParam
+    private Map<String, Integer> initialReplayIdMap;
```

Comment:
Is this all that is needed to set the value straight from the URI? https://github.com/apache/camel/blob/master/tooling/spi-annotations/src/main/java/org/apache/camel/spi/UriParam.java#L95